.github/workflows/nighly_fuzzing.yml

```diff
@@ -0,0 +1,56 @@
+name: keth-NIGHTLY-FUZZING
+
+on:
+  schedule:
+    - cron: 20 0 * * *
+  workflow_dispatch: {}
+
+permissions: read-all
+
+jobs:
+  tests-unit:
+    runs-on: ubuntu-latest-16-cores
+    timeout-minutes: 360
+    env:
+      PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION: python
+    steps:
+      - uses: actions/checkout@v4
+      - uses: astral-sh/setup-uv@v2
+        with:
+          enable-cache: true
+          cache-dependency-glob: uv.lock
+      - uses: actions/setup-python@v5
+        with:
+          python-version-file: ./cairo/.python-version
+      - name: Install Foundry
+        uses: foundry-rs/foundry-toolchain@v1
+        with:
+          version: nightly
+      - name: Run tests
+        id: tests
+        continue-on-error: true # To send a notification if the tests fail
```

Comment:
Seems to make the step green even if failed

Reply:
yes, hence the

```
      - name: Fail workflow if tests failed
        if: steps.tests.outcome == 'failure'
        run: exit 1
```

at the end